src/en/space-station-14/round-flow/antagonists/contracts-and-rep.md

```diff
@@ -0,0 +1,88 @@
+- Minor Theft Objectives
+  - Antagonist must steal minor items of a department and utilize syndicate fultons to extract them. 
+- Assist Traitors
+  - Ensure a traitor completes an contract. If the traitor rejects the contract, this objective will also reject, but will only take 5 minutes to refresh instead of the standard 15. 
+Moderate Risk Objectives are objectives that should get a traitor brigged for a while
+These objectives include but are not limited to;
+- High Value Theft Objectives
+  - Traitors must steal a high value object and use syndicate fultons to extract them.
+- Place Tracker Objective
+  - A syndicate tracker must be placed on a character person. Whether it be directly on them, in their pockets, or inside their bag. The tracker can be removed without uncompleting the contract
+- Corporate Espionage
+  - Utilizing a syndicate data-knife, a traitor must extract 1-3 researches from a science research console.
+High Risk objectives are objectives that should get a traitor brigged for a long time, permanently, or executed
+- Kill Targets
+  - The traitor must kill a target, this target can be a member of crew or another traitor. The target does not have to be gibbed. If the target is revived, the contract remains complete.
+- Extract Target
+  - a member of crew must be extracted using a syndicate fultons (MGSV style)
+- Corporate Sabotage
+  - A certain location or machine needs to be bombed using C4 or other high powered explosives
+- Die A Glorious Death
+
+## A Contractor's Tools
+
+Traitors recieve a new set of tools labeled "Objective" and "Station Destruction" respectively.
+These objective tools should never cost more then the contract value.
+
+Objective Items
+- Blood-Red Bug
+  - A bug that can be placed on walls, in plant pots, or even in food. Has no effect on crew. Should make a beeping noise and have a slight glow to show where it is. Is a tiny item
+- Tracker Bead
+  - A small bead which transmits its location should it be placed on a person. This person will always be on the crew monitor system as coords active. Is a tiny item
+- Syndicate Fultons
+  - Black Fultons with a red horizontal strip. Shouldnt require linking to any sort of beacon, rather it deletes the item, crate, and/or person thats been extracted. Is a medium sized item. 
+- Data Knife
+  - A knife with similar damage levels to the cybersun pen. Should be blatently obvious it is ~~a titanfall reference~~ contraband. Is a small item.
+
+Station Destruction Items
+A bit of a clarity, these items are mainly inspired by the existence of the "Delamination Crystal" from SS13. A purposeful way to cause a supermatter breach limited to antagonists which is extremely expensive.
+It also reduces EMAG functionality
+
+- The Anti-Matter Cascader
+  - When inserted into the AME Controller, allows the controller to go beyond the safety limits to generate an anti-matter explosion.
+- The Particle Hyper-Acceleration Module
```

Comment:
this is useless when you can just snip emitter wires. also ame being blown up is absolutely not the same as teslaloose

Reply:
the idea is that they would create powerful singulooses/teslalooses and much large AME explosions then without it. Of course you can still bomb the AME by removing shielding and you can still loose by cutting wires, but the idea of the Station Destruction Items is to allow you todo that with much larger effects (singuloosing a level 6 singularity, AME explosion being much large and powerful)